### DISCUSSION TITLE
[Feature] Filters for mobile app

### DISCUSSION THREAD

**Discussion by thefirstofthe300** [immich-app/immich#3751] [Feature Request]
### The feature

Due to the warning on the immich website stating not to use it as the sole method of backing up photos, my wife and I are uploading photos to both Immich and TheAppWhichShallNotBeNamed.

When we go through to delete what are essentially duplicate photos we've taken, we've found that the best way to handle doing so is to delete the duplicate from TheAppWhichShallNotBeNamed and then delete them from Immich by selecting all photos which are no longer on the device which requires sifting through all of the photos we've taken.

It would be nice if we could filter the photos in the app to only show photos which are not on the device to aid in our cleanup of duplicate photos.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2155
